src/handle.ml

```diff
@@ -77,6 +77,49 @@ let handle_tactic : sig_state -> Proof.t -> p_tactic -> Proof.t =
   | P_tac_print
   | P_tac_proofterm
   | P_tac_focus(_)      -> assert false (* Handled above. *)
+  | P_tac_assert(h,t)     ->
+     begin
```

Comment:
Using `begin` / `end` is not necessary here. Moreover, indentation is not correct (see other cases).